app/views/static_pages/videos.html.slim

```diff
@@ -7,7 +7,7 @@ section.main-content.container
           |  Motley Tones: Video Gallery
         p
           | Here are some videos provided by friends and fans. If you have any to share, please send the URL to us at
-          a<>[href="mailto:info@motleytones.com"]
+          = mail_to "info@motleytones.com", nil, subject: "Video to share!"
```

Comment:
A feature request you might want is to implement a contact form that emails you behind the scenes rather than putting the email address directly on the web.